person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services
+    @parent_permission && of_age?
```

Comment:
- [ ] Here `can_use_services` should return true if the person is of age or if they have permission from parents, kindly consider changing `and` (&&) to `or`